includes/Modules/Analytics_4.php

```diff
@@ -707,15 +707,6 @@ protected function get_datapoint_definitions() {
 			);
 		}
 
-		if ( Feature_Flags::enabled( 'conversionReporting' ) ) {
-			$datapoints['POST:clear-conversion-reporting-new-events']  = array(
-				'service' => '',
-			);
-			$datapoints['POST:clear-conversion-reporting-lost-events'] = array(
-				'service' => '',
-			);
-		}
```

Comment:
Old dismissal logic has been removed as part of this issue, since dismissing has been change to be user scoped.